src/adapters/rentals.ts

```diff
@@ -0,0 +1,16 @@
+import { RentalListing } from '@dcl/schemas'
+
+export type TileRentalListing = Pick<
+  RentalListing,
+  'expiration' | 'createdAt' | 'periods'
+>
+
+export function convertRentalListingToTileRentalListing(
+  rentalListing: RentalListing
+): TileRentalListing {
+  return {
+    expiration: rentalListing.expiration,
+    createdAt: rentalListing.createdAt,
+    periods: rentalListing.periods,
```

Comment:
Do we need all this information to be sent within the tiles? I think we should try to make the tile objects as tiny as possible with only what's necessary to render the whole map. Then if we need extra info (ie. for tooltips) we could fetch that on demand.. i'm worried about the impact of this on the size of the `/tiles` response.

Reply:
About the fields:
- After taking a look at it, the `createdAt` was removable.
- The expiration must be kept to be used when rendering the map, as it is used for the order.
- We're only showing the max price of the rental periods in the Popup in the UI, so in theory we would only need that, but, as the rental price might differ in the future with different periods, we might want to send all them, right?

Reply:
okay, lets remove the createdAt if it’s not necessary. In the future I would recommend not even sending the expiration and price of neither the sale listing nor rental listing, just a flag in order to render the tile with the right color. I would rather fetch that data on demand (ie. the popup fetches the listing data to render the price, etc). But that would require breaking changes to the current tiles, so I think we can move forward with this.

Reply:
After discussing which is the best thing to do, we've agreed on removing the `rentalListing` property from the legacy tile to return the `rentalPricePerDay` which will be the maximum price of the rental periods.
If we ever need more information out of the rental listing, we can use the Tiles V2 tiles by sending the rental listing id or other information about it.